phpstan.neon

```diff
@@ -211,6 +211,11 @@ parameters:
             message: '#Call to function is_string\(\) with float will always evaluate to false#'
             path: src/PhpParser/Printer/BetterStandardPrinter.php
 
+        # class_exists is forbidden to enforce static reflection, but at this point it works for us
```

Comment:
thats my interpretation of why class_exists() is/might be forbidden

Reply:
I can't see the line bellow this comment to comment on this :(

Reply:
I see. Better than "work for us" is to explain that class is a service used by Rector, so it must exist.

Reply:
What I initially meant: AbstractRector implements RectorInterface, therefore technically it would be enough to tell the class just needs to implement RectorInterface.

Mentioning AbstractRector is for convenience

Reply:
The "but at this point it works for us" comment should be changed to more exact one.